copy/copy.go

```diff
@@ -149,9 +148,12 @@ type Options struct {
 	// encrypted if non-nil. If nil, it does not attempt to decrypt an image.
 	OciDecryptConfig *encconfig.DecryptConfig
 
-	// MaxParallelDownloads indicates the maximum layers to pull at the same time.  A reasonable default is used if this is left as 0.
+	// MaxParallelDownloads indicates the maximum layers to pull at the same time. Applies to a single copy operation. A reasonable default is used if this is left as 0.
 	MaxParallelDownloads uint
 
```

Comment:
(Nit: Maybe drop this blank line to make the “`MaxParallelDownloads` is ignored” comment a bit more visible. And/or reverse the order so that newcomers read about `GlobalParallelDownloadsSemaphore` first.)